### ISSUE TITLE
Button Colors 

### ISSUE THREAD

**Issue by jminutaglio** [DDraiman1990/DroidTimeSelection#14]
The button colors for submit & cancel do not seem to be adjusting tint based on a style config:
```

        var style = HybridStyle()
        style.picker.titleColor = .white
        style.clock.indicatorColor = .purple
        style.modeButtonTint = .purple
        style.submitButtonColor = .green
        style.cancelButtonColor = .red
        
        
        let vc = DroidFactory.Hybrid.viewController(timeFormat: .twelve, style: style)
        
        vc.selector.onCancelTapped = {
            vc.dismiss(animated: true, completion: nil)
        }
                
        vc.selector.onOkTapped = {    
            let value = vc.selector.time //the time selected
            print("selected time:", value)
                        
            vc.dismiss(animated: true, completion: nil)
        }
                
        vc.selector.onSelectionChanged = { [weak self] value in
        }
        
        present(vc, animated: true, completion: nil)


```

Results in:
<img width="344" alt="Screen Shot 2021-03-28 at 11 59 17 PM" src="https://user-images.githubusercontent.com/49879291/112785590-56886800-9022-11eb-82fd-c26806a3ca76.png">

**Comment by DDraiman1990** [comment]
Whoops. Will address this tonight

**Comment by DDraiman1990** [comment]
Fixed in version 2.0.1 which is now live. Thanks for bringing this up.
